framework/auth/core.py

```diff
@@ -285,8 +285,22 @@ class User(GuidStoredObject, AddonModelMixin):
     #              'expiration': <datetime>}
     # }
 
-    # email lists to which the user has chosen a subscription setting
+    # TODO remove this field once migration (scripts/migration/migrate_mailing_lists_to_mailchimp_fields.py)
+    # has been run. This field is deprecated and replaced with mailchimp_mailing_lists
     mailing_lists = fields.DictionaryField()
+
+    # email lists to which the user has chosen a subscription setting
+    mailchimp_mailing_lists = fields.DictionaryField()
+    # Format: {
+    #   'list1': True,
+    #   'list2: False,
+    #    ...
+    # }
+
+    osf_mailing_default = {settings.OSF_HELP_LIST: True}
+
+    # email lists to which the user has chosen a subscription setting, being sent from osf, rather than mailchimp
+    osf_mailing_lists = fields.DictionaryField(default=osf_mailing_default)
```

Comment:
As mentioned in a previous comment, you should not use a mutable value for `default`. MODM allows you to pass a callable that will return the default value upon instantiation. So you could pass a `lambda` that returns your default dict value